pyproject.toml

```
@@ -31,7 +31,10 @@ numpy = [
     { version = ">=1.26.1", python = ">=3.9,<3.13" },
 ]
 python = "^3.7"
-pint = ">=0.10.0"
+pint =[
+    { version = ">=0.10", python = "3.8" },
```

```suggestion
pint = [
    { version = ">=0.10", python = ">=3.7,<3.9" },
```
hopefully this appeases it. no pint was getting installed for the py37 CI lanes